L/LLVMExtra/build_tarballs.jl

```diff
@@ -0,0 +1,73 @@
+using BinaryBuilder, Pkg
+using Base.BinaryPlatforms
+
+include("../../fancy_toys.jl")
+
+name = "LLVMExtra"
+repo = "https://github.com/vchuravy/LLVMExtra.git"
+version = v"0.0.1"
+
+# Collection of sources required to build attr
+sources = [GitSource(repo, "30882a8aadea1e15178e6fc8a70165a88f8377b0")]
+
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd LLVMExtra
+CMAKE_FLAGS=()
+# Release build for best performance
+CMAKE_FLAGS+=(-DCMAKE_BUILD_TYPE=RelWithDebInfo)
+# Install things into $prefix
+CMAKE_FLAGS+=(-DCMAKE_INSTALL_PREFIX=${prefix})
+# Explicitly use our cmake toolchain file and tell CMake we're cross-compiling
+CMAKE_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN})
+CMAKE_FLAGS+=(-DCMAKE_CROSSCOMPILING:BOOL=ON)
```

Comment:
Is this really needed?  We should already be setting this variable correctly: https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/5684f204cc503b746a032536b90a0667745589bd/src/BuildToolchains.jl#L36-L45

Reply:
I stole what we have for Enzyme, so it might also work